bool led_state
bool emg_stop
bool epos_reset
bool pc
bool laser
bool video_server